neuralmonkey/decoders/autoregressive.py

```diff
@@ -185,7 +189,12 @@ def embedding_matrix(self) -> tf.Variable:
     def get_logits(self, state: tf.Tensor) -> tf.Tensor:
         """Project the decoder's output layer to logits over the vocabulary."""
         state = dropout(state, self.dropout_keep_prob, self.train_mode)
-        return tf.matmul(state, self.decoding_w) + self.decoding_b
+
+        logits = tf.matmul(state, self.decoding_w)
+        if self.decoding_b is not None:
+            logits += self.decoding_b
```

Comment:
Nemůžou bejt `decoding_b` místo `None` prostě `tf.zeros`? to by tu pak odpadl tenhle `if`.